# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ScriptLoader.h
# Commit: a48aff5f2555
# Full Hash: a48aff5f2555af61a9bc1ee0d77f6058f47d0501
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 7e54a223001d -r a48aff5f2555 dom/script/ScriptLoader.h
--- a/dom/script/ScriptLoader.h	Thu Feb 17 15:57:51 2022 +0000
+++ b/dom/script/ScriptLoader.h	Thu Feb 17 15:57:52 2022 +0000
@@ -56,14 +56,18 @@
 class AutoJSAPI;
 class DocGroup;
 class Document;
-class ModuleLoader;
 class LoadedScript;
 class ScriptLoaderInterface;
+class ModuleLoader;
 class ModuleLoadRequest;
 class ModuleScript;
 class SRICheckDataVerifier;
 class SRIMetadata;
 class ScriptLoadHandler;
+class ScriptLoadRequest;
+class ScriptLoadRequestList;
+class ScriptLoadContext;
+
 class ScriptLoader;
 class ScriptRequestProcessor;
 